lib/travis/logs/config.rb

```diff
@@ -8,7 +8,7 @@ class Config < Travis::Config
              logs_database: { adapter: 'postgresql', database: "travis_logs_#{Travis.env}", encoding: 'unicode', min_messages: 'warning' },
              s3:            { hostname: 'archive.travis-ci.org', access_key_id: '', secret_access_key: '', acl: :public_read },
              pusher:        { app_id: 'app-id', key: 'key', secret: 'secret', secure: false },
-             sidekiq:       { namespace: 'sidekiq', pool_size: 3 },
+             sidekiq:       { namespace: 'sidekiq', pool_size: 22 },
```

Comment:
It is a big jump. It turns out that in staging it's not been able to start up for some time because the `pool_size` required for the concurrency we pass it is anywhere from 7 to 22 (concurrency is set to 20 and then + 2) and it defaults to 3. As I understand it, we set a lot of these in production via config values. But in Staging they weren't set. Once I get this PR to actually succeed, I'll probably split this change out into a separate PR since it's worth noting.